src/Storages/StorageKeeperMap.cpp

```diff
@@ -623,6 +640,302 @@ NamesAndTypesList StorageKeeperMap::getVirtuals() const
         {std::string{version_column_name}, std::make_shared<DataTypeInt32>()}};
 }
 
+namespace
+{
+
+constexpr std::string_view backup_data_filename = "data.bin";
+constexpr std::string_view backup_data_location_filename = "data_location.bin";
```

Comment:
It's just a text file containing a path, doesn't it? Let's rename it to `data_location.txt` then?